lib/Wikia/src/Service/User/Permissions/PermissionsServiceImpl.php

```diff
@@ -88,6 +89,27 @@ public function getAutomaticGroups( \User $user, $recacheAutomaticGroups = false
 	}
 
 	/**
+	 * Returns list of access control groups that need to be added based on the explicit user groups
+	 *
+	 * @param \User $user
+	 * @return array
+	 */
+	private function getAutomaticAccessControlGroups( \User $user ) {
+		if ($user->getId()) {
```

Comment:
This is exactly how it is checked a few lines above and I tried to be consistent. Should I change it in both places?